podcasts/Onboarding/Import/ImportDetailsView.swift

```diff
@@ -1,56 +1,134 @@
                         .foregroundColor(AppTheme.color(for: .primaryText01, theme: theme))
                         .fixedSize(horizontal: false, vertical: true)
 
                     appInstructions
 
+                    if importSource.id == .opmlFromURL {
+                        opmlImportView
+                    }
+
                     Spacer()
                 }.padding([.leading, .trailing], Constants.horizontalPadding)
             }
 
             // Hide button for other
-            if app.id != .other {
-                Button(app.id == .applePodcasts ? L10n.importInstructionsInstallShortcut : L10n.importInstructionsOpenIn(app.displayName)) {
-                    viewModel.openApp(app)
+            if !importSource.hideButton {
+                if importSource.id == .opmlFromURL {
+                   opmlViewButton
+                } else {
+                    Button(importSource.id == .applePodcasts ? L10n.importInstructionsInstallShortcut : L10n.importInstructionsOpenIn(importSource.displayName)) {
+                        viewModel.openApp(importSource)
+                    }
+                    .buttonStyle(RoundedButtonStyle(theme: theme))
+                    .padding([.leading, .trailing], Constants.horizontalPadding)
                 }
-                .buttonStyle(RoundedButtonStyle(theme: theme))
-                .padding([.leading, .trailing], Constants.horizontalPadding)
             }
         }.padding(.top, 16).padding(.bottom)
-        .background(AppTheme.color(for: .primaryUi01, theme: theme).ignoresSafeArea())
+            .background(AppTheme.color(for: .primaryUi01, theme: theme).ignoresSafeArea())
     }
 
     @ViewBuilder
     private var appInstructions: some View {
-        let lines = app.steps.split(separator: "\n").map { String($0).trim() }
+        let lines = importSource.steps.split(separator: "\n").map { String($0).trim() }
         VStack(alignment: .leading, spacing: 20) {
-           ForEach(lines, id: \.self) { line in
-               Text(line)
-                   .font(style: .subheadline, maxSizeCategory: .extraExtraExtraLarge)
-                   .foregroundColor(AppTheme.color(for: .primaryText01, theme: theme))
-                   .fixedSize(horizontal: false, vertical: true)
-           }
-       }
+            ForEach(lines, id: \.self) { line in
+                Text(line)
+                    .font(style: .subheadline, maxSizeCategory: .extraExtraExtraLarge)
+                    .foregroundColor(AppTheme.color(for: .primaryText01, theme: theme))
+                    .fixedSize(horizontal: false, vertical: true)
+            }
+        }
+    }
+
+    @ViewBuilder
+    private var opmlImportView: some View {
+        VStack {
+            TextField("https://...", text: $opmlURLText)
+                .textFieldStyle(RoundedBorderTextFieldStyle())
+                .autocapitalization(.none)
```

Comment:
Wdyt about defaulting to the URL keyboard type?

```suggestion
                .autocapitalization(.none)
                .keyboardType(.URL)
```